curriculum/challenges/english/blocks/lab-pricing-plans-layout/68e45a2b4ab9b2f48b1cc6df.md

```diff
@@ -0,0 +1,414 @@
+---
+id: 68e45a2b4ab9b2f48b1cc6df
+title: Design a Pricing Plans Layout Page
+challengeType: 25
+dashedName: design-a-pricing-plans-layout-page
+demoType: onClick
+---
+
+# --description--
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab.
+
+**User Stories**
+
+1. Your page should have a `h1` element with the text `Pricing Plans`.
+2. Your page should have a `div` element with the class `pricing-container`.
+    - It should have a `display` property and the `flex` value.
+    - It should have a `flex-wrap` property and the `wrap` value.
+3. Your page should have three `div` elements with the class `pricing-card` to represent the pricing plans.
+    - One of your `div` elements should have the class `basic-plan` in addition to the `pricing-card` class.
+    - One of your `div` elements should have the class `pro-plan` in addition to the `pricing-card` class.
+    - One of your `div` elements should have the class `premium-plan` in addition to the `pricing-card` class.
+4. Your `basic-plan` element should have an `h2` element with the text `Basic Plan`.
+5. Your `basic-plan` element should have a `p` element with the text `$9/month`.
+6. Your `pro-plan` element should have an `h2` element with the text `Pro Plan`.
+7. Your `pro-plan` element should have a `p` element with the text `$19/month`.
+8. Your `premium-plan` element should have an `h2` element with the text `Premium Plan`.
+9. Your `premium-plan` element should have a `p` element with the text `$29/month`.
+10. Each of your `.pricing-card` elements should use Flexbox to stack its content in a column and justify the space between the children using `space-between`. Each element should also set the `flex` property to `0 0 200px` to give it a consistent width and prevent it from growing or shrinking in the layout. Each element should set the `border` property to `2px solid black` to see how the different cards take up space.
+11. The `basic-plan` element should appear first in the layout. You should use the `order` property for this.
+12. The `pro-plan` element should appear second in the layout. You should use the `order` property and set its `flex-grow` property to `2` so it takes up more space than the other plans.
+13. The `premium-plan` element should come last in the layout. You should use the `order` property for this.
+
+**Note:** Be sure to link your stylesheet in your HTML and apply your CSS.
+
+# --hints--
+
+Your `h1` element should contain the text `Pricing Plans`.
+
+```js
+const h1 = document.querySelector('h1');
+assert.strictEqual(h1?.textContent, 'Pricing Plans');
+```
+
+Your `div` element should contain the class `pricing-container`.
+
+```js
+const div = document.querySelector('.pricing-container');
+assert.strictEqual(div?.tagName, 'DIV');
+```
+
+Your `pricing-container` element should contain the property `display` and the value `flex`.
```

Comment:
```suggestion
Your `.pricing-container` selector should have the property `display` and the value `flex`.
```

you are testing specifically for the selector. To test that the element has a certain styling you would need to use `getComputedStyle`

https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle